qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -646,6 +647,23 @@ def _unroll_condition(property_set):
                 else _opt + _unroll_if_out_of_basis + _depth_check + _size_check
             )
             optimization.append(DoWhileController(opt_loop, do_while=_opt_control))
+
+            if optimization_level == 3 and pass_manager_config.coupling_map:
+                vf2_call_limit, vf2_max_trials = common.get_vf2_limits(
+                    optimization_level,
+                    pass_manager_config.layout_method,
+                    pass_manager_config.initial_layout,
+                )
+                optimization.append(
+                    VF2PostLayout(
+                        target=pass_manager_config.target,
+                        seed=pass_manager_config.seed_transpiler,
```

Comment:
I would set this to -1 to disable randomization:

```suggestion
                        seed=-1,
```

The details on why this is preferable here: https://github.com/Qiskit/qiskit/pull/13492 but tl;dr is doing randomization here just makes the problem here harder so we disabled it everywhere else.